.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true
+  },
+  extends: 'standard',
+  overrides: [
+    {
+      env: {
+        node: true,
+        browser: true,
+        es2021: true,
+        jest: true,
+        'cypress/globals': true
+      },
+      files: [
+        '.eslintrc.js', '**/*.test.js', '**/*.cy.js'
```

Comment:
Instead of merging both overrides together, you should have two seperate blocks for jest and cypress

Reply:
noted :)